jetstream/test/jetstream_test.go

```diff
@@ -431,6 +431,236 @@ func TestCreateStreamMirrorCrossDomains(t *testing.T) {
 	}
 }
 
+func TestCreateOrUpdate_CreateStream(t *testing.T) {
```

Comment:
Can you refactor this to have a single test for this function (i.e. instead of having `TestCreateOrUpdate_CreateStream` and `TestCreateOrUpdate_UpdateStream` to just have `TestCreateOrUpdateStream`.

Reply:
yes, mentioned tests has been merged together